flink-kubernetes-operator/pom.xml

```diff
@@ -198,6 +198,13 @@ under the License.
             <artifactId>junit-jupiter-params</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Please avoid using Mockito as per the Flink coding guidelines: https://flink.apache.org/how-to-contribute/code-style-and-quality-common/#avoid-mockito---use-reusable-test-implementations

I suggest overriding the getPodList method of the standalone flink service by hand or using the TestingFlinkService.